pbf/models.py

```diff
@@ -286,7 +286,9 @@ def get_play_cost(self):
         return sum([card.cost for card in self.play.all()])
 
     def get_gain_energy(self):
-        amount = max([self.starting_energy] + [p.get_energy() for p in self.presence_set.all()]) + sum([p.get_plus_energy() for p in self.presence_set.all()])
```

Comment:
removing the reference to `starting_energy` here is incorrect behaviour. This nerfs all spirits that have energy gain per turn to instead start with 0 energy gained per turn. For example, with this change, try starting a game with Roots. Roots should gain 1 energy per turn, but with this change, now it gains 0 energy per turn, which is incorrect. `starting_energy` needs to remain here.

Reply:
I understand that this change was meant to fix the problem where Wounded Waters Bleeding gains 4 energy per turn even when it is not entitled to, and I sympathise that this is a real problem. The correct fix is to set starting_energy for Wounded Waters Bleeding to its correct value of 0 (as it starts out gaining 0 energy per turn), not to remove starting_energy from the energy gain calculation.

Giving Wounded Waters Bleeding 4 energy to start the game needs to be done by setting its `energy` to 4 upon creation, not `starting_energy`.